exchanges/request/request_types.go

```diff
@@ -23,12 +23,14 @@ const (
 var (
 	MaxRequestJobs   = DefaultMaxRequestJobs
 	MaxRetryAttempts = DefaultMaxRetryAttempts
+	GlobalReporter   Reporter
```

Comment:
I would prefer this to be unexported and you instead have a setup function

Reply:
`request.SetupGlobalReporter` as been added to this end